Undefined name 'a' on line 11 of rpi-detector.py

**Describe the bug**
Fixed in #55

A clear and concise description of what the bug is and what program(s) has the bug.

$ `flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics`
```
./PyPrograms/raspberry-pi-detector/rpi-detector.py:11:33: F821 undefined name 'a'
ping = os.system("ping -c 1 " + a) # Ping the local address raspberrypi.local on the LAN network
                                ^
1     F821 undefined name 'a'
1
```

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]
 - User Permission Level: [e.g. Admin]
 - Python Version [e.g. 3.1]

**Additional context**
Add any other context about the problem here.

--- a/.github/workflows/lint_python.yml
+++ b/.github/workflows/lint_python.yml
@@ -1,0 +1,27 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit black codespell flake8 flake8-2020 flake8-bugbear
+                         flake8-comprehensions isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101,B301,B403,B605 .
+      - run: black --check . || true
+      - run: codespell || true  # --ignore-words-list="" --skip="*.css,*.js,*.lock"
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88
+                      --show-source --statistics
+      - run: isort --check-only --profile black . || true
+      - run: pip install -r requirements.txt || pip install --editable . || pip install . || true
+      - run: mkdir --parents --verbose .mypy_cache
+      - run: mypy --ignore-missing-imports --install-types --non-interactive . || true
+      - run: pytest . || true
+      - run: pytest --doctest-modules . || true
+      - run: shopt -s globstar && pyupgrade --py36-plus **/*.py || true
+      - run: safety check

--- a/PyPrograms/raspberry-pi-detector/rpi-detector.py
+++ b/PyPrograms/raspberry-pi-detector/rpi-detector.py
@@ -5,6 +5,7 @@
 #
 
 import os
+a = "127.0.0.1"  # TODO: Change this to another server on your local network.
 print("Raspberry Pi Detector")
 print("This uses the GPL V3 Licence. Read the file LICENCE to see it.")
 ip = "raspberrypi.local" # Name of the default Raspberry Pi LAN Hostname

